tools/ci_build/requirements.txt

```diff
@@ -5,3 +5,6 @@ numpy==1.24.0
 coloredlogs==15.0
 transformers==4.24.0
 psutil
+
+# package used by smooth quant test
+neural-compressor>=2.2
```

Comment:
Do not add it here. This is for transformer test.

For example, for Linux GPU pipeline, you can see 
https://github.com/microsoft/onnxruntime/blob/2ae041f390917f776b4bb746fe9570f047c274d9/tools/ci_build/github/azure-pipelines/linux-gpu-ci-pipeline.yml#L147

So need update here:
tools/ci_build/github/linux/docker/scripts/manylinux/requirements.txt